samples/multi-yaml/fe/tye.yaml

```diff
@@ -0,0 +1,8 @@
+app: multi-tye # name of the bigger application
+name: frontend # name of this application
+services:
+- name: frontend
+  project: frontend/frontend.csproj
+  replicas: 2
+dependencies:
+- path: ../be/tye.yaml
```

Comment:
Related to #239 - this is an improvement, but doesn't solve most of the really valuable scenarios.

The thing that gives me pause, I'm not sure why you'd want to break up tye.yaml into multiple files if you have to have access to them all at once.

Reply:
This is just a sample to make sure it works as advertised.